docs/getting-started.md

```diff
@@ -346,445 +252,363 @@ const ctype = Kilt.CType.fromSchema({
-  message: '0xFEED...CAFE',
-  nonce: '0x8f18d3394f4d325106c7b618046f6a8415bff1d5b4d267a8',
-  hash: '0x4f4108cf390eda665315cbff7cc21c155ae5895918a8691b04e0c27b803c3bc8',
-  signature: '0x01ee177a0ce94603de8958f854d7cec84adc09f6f5ec400e1432dfe6cf69418174b8c841f92664dd7c89818560aafee747e453200dab8a47b39a5fdfa3b4b3d880'
-}
-```
-
-After receiving the message, the Claimer just needs to save it and use it later for verification:
-
-```typescript
-let myCredential: Credential
-if (
-  messageBack.body.type === Kilt.Message.BodyType.SUBMIT_ATTESTATION
-) {
-  myCredential = Kilt.Credential.fromCredential({
-    ...messageBack.body.content,
-    request: requestForAttestation,
   })
-}
-```
-
-## 6. Verify a Claim
 
-As in the attestation, you need a second account to act as the verifier. The verifier only needs a light DID [see claimer DID](#22-generate-a-light-did-for-the-claimer):
-
-```typescript
-const verifierMnemonic = generateMnemonic()
-const verifierLightDid = new Kilt.Did.LightDidDetails({
-  authenticationKey: {
-    publicKey: verifierSigningKeypair.publicKey,
-    type: Kilt.Did.DemoKeystore.getKeypairTypeForAlg(
-      verifierSigningKeypair.alg
-    ),
-  },
-  encryptionKey: {
-    publicKey: verifierEncryptionKeypair.publicKey,
-    type: Kilt.Did.DemoKeystore.getKeypairTypeForAlg(
-      verifierEncryptionKeypair.alg
-    ),
-  },
+/* Create an off-chain DID with the generated keys. */
+const verifierLightDid = Kilt.Did.LightDidDetails.fromDetails({
+  authenticationKey: verifierAuthenticationKey,
+  encryptionKey: verifierEncryptionKey,
 })
+console.log(`Verifier DID: ${verifierLightDid.did}`)
 ```
 
-Before a claimer sends any data to a verifier, the verifier needs to initiate the verification process by requesting a presentation for a specific CTYPE.
-Therefore, the verifier knows which properties are included in the credential.
-A claimer can also hide selected properties from their credential.
-This is an **important feature for the privacy of a claimer** as this enables them to only show necessary properties for a specific verification.
+### 5.2 Ask for credentials
 
-### 6.1. Request presentation for CTYPE
+A verifier usually requires a claimer to fulfil some requirements before providing a specific service.
+The verifier needs to initiate the verification process by requesting a presentation for a specific CType.
+In this case, the verifier wants to verify the name of the claimer.
+To do so, it will send a request for credential message to the claimer.
 
 ```typescript
-const messageBodyForClaimer: MessageBody = {
-  type: Kilt.Message.BodyType.REQUEST_CREDENTIAL,
-  content: { ctypes: [ctype.hash] },
-}
-const messageForClaimer = new Kilt.Message(
-  messageBodyForClaimer,
+const requestForCredentialMessage = new Kilt.Message(
+  {
+    type: Kilt.Message.BodyType.REQUEST_CREDENTIAL,
+    content: {
+      cTypes: [
+        { cTypeHash: ctype.hash, trustedAttesters: [attesterFullDid.did] },
+      ],
+    },
+  },
```

Comment:
For some reason this comment didn't make it... I was asking if we don't allow for a challenge on the request, or have some other challenge procedure for verification at this point?

Reply:
https://github.com/KILTprotocol/sdk-js/pull/477/commits/8cd592eaf1f35342a10fa3053be0878f8ffcecaf